model/change_point_detection.go

```diff
@@ -389,14 +388,44 @@ func GetTotalPagesForChangePointsGroupedByVersion(ctx context.Context, env cedar
 		}
 		return int(math.Ceil(float64(res.Count) / float64(pageSize))), nil
 	}
-	return 0, errors.New("Not able to get count of total changepoints matching query")
+	return 0, nil
 }
 
-func appendAfterBaseGetChangePointsByVersionAgg(projectId string, additionalSteps ...bson.M) []bson.M {
+func appendAfterBaseGetChangePointsByVersionAgg(projectId, variantRegex, versionRegex, taskRegex, testRegex, measurementRegex string, threadLevels []int, additionalSteps ...bson.M) []bson.M {
+	matchStage := bson.M{
```

Comment:
should you be checking if the regexs are empty before adding them to the match stage?

Reply:
The empty regex does nothing regardless.